src/replicache-options.ts

```diff
@@ -81,9 +81,18 @@ export interface ReplicacheOptions<MD extends MutatorDefs> {
    * Replicache will also log `'info'` and `'error'` messages. When set to
    * `'info'` we log `'info'` and `'error'` but not `'debug'`. When set to
    * `'error'` we only log `'error'` messages.
+   * Default is `'info'`.
    */
   logLevel?: LogLevel;
 
+  /**
+   * Enables custom handling of logs. By default logs are logged to the console.
+   * If you would like logs to be sent somewhere else (e.g. to a cloud logging
+   * service like DataDog) you can provide a custom `LogSink`.  Logs at or above
+   * [[ReplicacheOptions.logLevel]] are sent to this `LogSink`.
+   */
+  logSink?: LogSink;
```

Comment:
The thing that irritates me is that I think everyone will _always_ want the logs to go to console, regardless of whether they also go to DataDog. I can't think of a reason you would _not_ want them to go to console.

OTOH, providing a sink API and *not* providing a way to get rid of the console seems like a weird API.

What if instead we called this `logSinks` and it was an array of `LogSink` and by default it was the console logger. But people could `push()` on the datadog logger too if they wanted (which would be the typical usage of this feature).

Reply:
@arv

Reply:
I don't know. If I'm deploying gmail I'd be more than happy to not log to console by default. I would want more control.

I think, the idea to use an array is a bit too clever... and also, I don't like the mutability of options.

The `LogSink` is such a small interface that I think it is cleaner to let people do their own `TeeLogger`.

Reply:
Updated to logSinks array.  what do you all think?

Reply:
I also updated reflect to take an array of LogSinks for consistency https://github.com/rocicorp/reflect/pull/104